### PR TITLE
PSMDB-352: Release notes - v3.4.20-2.18

### PR DESCRIPTION
**Pull request by borys-belinsky**

### CODE REVIEW

File: source/conf.py [hunk 1]
```diff
@@ -209,7 +209,7 @@
 # If true, "Created using Sphinx" is shown in the HTML footer. Default is True.
 #html_show_sphinx = True
 
-# If true, "(C) Copyright ..." is shown in the HTML footer. Default is True.
+# If true, "(C) Copyripght ..." is shown in the HTML footer. Default is True.
```
Comment (tplavcic) [inline source/conf.py]: Typo?

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: source/conf.py (hunk 1 above)
Comment (borys-belinsky) [reply source/conf.py]: It is a comment which will not appear anywhere, you can safely ignore it

File: source/release_notes/3.4.20-2.18.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+several enterprise-grade features:
+
+- `External Authentication <https://www.percona.com/doc/percona-server-for-mongodb/3.4/authentication.html>`_
+- `Audit Logging <https://www.percona.com/doc/percona-server-for-mongodb/3.4/audit-logging.html>`_
+- `Profiling Rate Limiting <https://www.percona.com/doc/percona-server-for-mongodb/3.4/rate-limit.html>`_
+- `Hot Backup <https://www.percona.com/doc/percona-server-for-mongodb/3.4/hot-backup.html>`_
+- `Log Redaction <https://www.percona.com/doc/percona-server-for-mongodb/3.4/log-redaction.html>`_
+
+|PSMDB| requires no changes to MongoDB applications or
+code. This release is based on `MongoDB 3.4.19
```
Comment (igorsol) [inline source/release_notes/3.4.20-2.18.rst]: version is wrong. Should be 3.4.20

**Review by igorsol — CHANGES_REQUESTED** [review]

File: source/release_notes/3.4.20-2.18.rst (hunk 2 above)
Comment (borys-belinsky) [reply source/release_notes/3.4.20-2.18.rst]: Fixed. the correction also applies to the URL

**Review by tplavcic — APPROVED** [review]
LGTM

**Review by igorsol — APPROVED** [review]

**Pull request merged**